lib/_tls_common.js

```diff
@@ -93,9 +93,7 @@ function SecureContext(secureProtocol, secureOptions, minVersion, maxVersion) {
   }
 }
 
-function createSecureContext(options) {
-  if (!options) options = kEmptyObject;
-
+function createSecureContext(options = kEmptyObject) {
```

Comment:
This change is not same as the change you provide.

Reply:
When implementing this, there a few spots where they could be optimized a bit further, such as this. 

I'm happy to undo, as there is no lint rule regarding those changes, but I think they are relevant enough to include.

Reply:
if there is no lint rule, this is mostly a cosmetic change. please refer to the contributing guideline. it is recommended to include eslint rule for every cosmetic change that can be reverted. persisting code styling is important.

Reply:
I'll make sure to read it over :sweat_smile: